src/info/author.rs

```diff
@@ -51,7 +51,7 @@ impl std::fmt::Display for Author {
         } else {
             write!(
                 f,
-                "{}% {} {}",
+                "{:>3}% {} {}",
```

Comment:
With a systematic left pad of 3, we introduce extra spaces in most cases, misaligning this info line compared to the others.

 I think there are three cases:

1. Only one contributor at 100% -> no left pad
2. Multiple contributors with a top contributor having a 2-digit contribution percentage -> left pad of size 2
3. Less common scenario: multiple contributors, but none of them has a contribution percentage higher than 9% -> no left pad

Reply:
Ah, I see. In other words, when aligning, the first author shouldn't have extra spaces between it and the stat label?
So, to give another example, with 95% and 5%, the first author shouldn't have any alignment, and the second author should be aligned 1 extra space to fill in for the missing digit?

Reply:
While you're point is totally valid, I might argue that eliminating the 1 space is not worth the additional complexity.

In particular, we cannot simply tweak the std::fmt::Display, like this, we'd need a whole different way to dynamically format based on the vector of authors.

Perhaps this could be done in the std::fmt::Display for AuthorsInfo, but definitely would be more complex.

For reference, it looks like this:
```
❯ cargo run --release
    Finished release [optimized] target(s) in 0.15s
     Running `target/release/onefetch`
                 R RR RR                  Luke Hsiao ~ git version 2.42.0
              R RRRRRRRR R          R     -------------------------------
 R RR       R RRRRRRRRRRRRR R      RR     Project: onefetch (5 branches, 44 tags)
rR RRR    R RRRRRRRRRRRRRRRRR R   RRR R   Description: Command-line Git information tool
RRR RR   RRRRRRRRRRRRRRRRRRRRRRR  RRRRR   HEAD: a907d352a019 (main, fork/main)
 RRRRR  RRRRRRRRRRRRRRRRRRRRRRRR  RRRR    Version: 2.18.1
  RRR RRRRRRRRRRRRRRRRRRRRRRRRRRRR RR     Created: 5 years ago
    R  RRRRRRRRRR=  RR = RRRRRRRRRRR      Languages:
     RRRRRRRRRRRR=  RR = RRRRRRRRRR                  ● Rust (97.5 %) ● Shell (2.1 %)
      RRRRRRRRRRR   RR   RRRRRRRRRR                  ● Makefile (0.4 %)
     RR==RRRRRRRRRRRRRRRRRRRRRR===RR      Dependencies: 29 (Cargo)
     RR =  ==RRRRRRR  RRRRRR==  = RR      Authors:  52% Ossama Hjaji 1212
      RR =     ===========     = RR                 21% dependabot[bot] 485
       RR                        R                   4% Spenser Black 88
        R                       R         Last change: 10 hours ago
         R                                Contributors: 136
                                          URL: ssh://git@github.com/o2sh/onefetch.git
                                          Commits: 2309
                                          Churn (246): Cargo.lock 105
                                                       Cargo.toml 76
                                                       …/vercel/package-lock.json 23
                                          Lines of code: 5050
                                          Size: 2.45 MiB (144 files)
                                          License: MIT


```

So, while there is an extra space, IMO it is not visually distracting.

Reply:
Nope, I take this back. Turns out the std::fmt::Display of AuthorsInfo makes this pretty trivial to do.

I've updated the commit with the new approach, which now looks like this

```
❯ cargo run --release
   Compiling onefetch v2.18.1 (/home/lukehsiao/repos/github.com/o2sh/onefetch)
    Finished release [optimized] target(s) in 4.95s
     Running `target/release/onefetch`
                 R RR RR                  Luke Hsiao ~ git version 2.42.0
              R RRRRRRRR R          R     -------------------------------
 R RR       R RRRRRRRRRRRRR R      RR     Project: onefetch (5 branches, 44 tags)
rR RRR    R RRRRRRRRRRRRRRRRR R   RRR R   Description: Command-line Git information tool
RRR RR   RRRRRRRRRRRRRRRRRRRRRRR  RRRRR   HEAD: a907d352a019 (main, fork/main)
 RRRRR  RRRRRRRRRRRRRRRRRRRRRRRR  RRRR    Pending: 1+-
  RRR RRRRRRRRRRRRRRRRRRRRRRRRRRRR RR     Version: 2.18.1
    R  RRRRRRRRRR=  RR = RRRRRRRRRRR      Created: 5 years ago
     RRRRRRRRRRRR=  RR = RRRRRRRRRR       Languages:
      RRRRRRRRRRR   RR   RRRRRRRRRR                  ● Rust (97.5 %) ● Shell (2.1 %)
     RR==RRRRRRRRRRRRRRRRRRRRRR===RR                 ● Makefile (0.4 %)
     RR =  ==RRRRRRR  RRRRRR==  = RR      Dependencies: 29 (Cargo)
      RR =     ===========     = RR       Authors: 52% Ossama Hjaji 1212
       RR                        R                 21% dependabot[bot] 485
        R                       R                   4% Spenser Black 88
         R                                Last change: 10 hours ago
                                          Contributors: 136
                                          URL: ssh://git@github.com/o2sh/onefetch.git
                                          Commits: 2309
                                          Churn (213): Cargo.lock 100
                                                       Cargo.toml 71
                                                       …/vercel/package-lock.json 21
                                          Lines of code: 5087
                                          Size: 2.45 MiB (144 files)
                                          License: MIT

```